src/utils/helpers/poller/poller.js

```diff
@@ -41,45 +41,49 @@ import './../../promises';
  */
 export default (queryOptions, functions, options) => {
   if (!setupValid(queryOptions, functions)) {
-    return;
+    return undefined;
```

Comment:
right, I was wondering if we wanted to return `undefined`. I suppose in this context, it's the best option